src/astro/sql/operators/drop.py

```diff
@@ -1,13 +1,14 @@
 from typing import Dict
 
 from airflow.decorators.base import get_unique_task_id
-from airflow.models import BaseOperator
+from airflow.models.baseoperator import BaseOperator
 
 from astro.databases import create_database
+from astro.sql.operators.base_operator import AstroSQLBaseOperator
 from astro.sql.table import Table
 
 
-class DropTableOperator(BaseOperator):
+class DropTableOperator(AstroSQLBaseOperator, BaseOperator):
```

Comment:
```suggestion
class DropTableOperator(AstroSQLBaseOperator):
```
We don't need that as `AstroSQLBaseOperator` is already based on `BaseOperator`.